packages/fluentui/react-bindings/src/hooks/useIFrameFocusDispatch.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+export const FUI_FRAME_EVENT = 'fuiframefocus';
+
+/**
+ * Watch the `document.activeElement` and dispatch a custom event if the current active element is a iframe.
+ */
+export const useIFrameFocusDispatch = (
+  enableFrameFocusDispatch: boolean,
+  targetDocument: Document,
+  pollDuration: number = 1000,
+) => {
+  const timeoutRef = React.useRef<ReturnType<typeof setTimeout>>();
+
+  React.useEffect(() => {
+    if (enableFrameFocusDispatch) {
+      timeoutRef.current = setInterval(() => {
```

Comment:
if targetDocument is provided, use it for creating interval (something like targetDocument.setInterval)

Reply:
document cannot call `setInterval` it is a window global method. If you really wanted to do smth like that it would have to be:

```js
document.defaultView.setInterval
```

Reply:
Yeah, I was trying to use global `window` before, till @layershifter told me it would be safe enough to just call `setInterval`